### PR TITLE
Add lw 2 solution

### PR DESCRIPTION
**Pull request by eqxba**
Responsive page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            <a href="http://blazrobar.com/" class="galery-link galery-item">
+                <img src="Galery-link.jpg">
+                <i class="fas fa-angle-right"></i>
+            </a>
+            <img src="Bridge.jpg" class="galery-item">
+            <img src="Woman.jpg" class="galery-item">
+            <div class="galery-breaker">
+            </div>
+        </div>
+        <div class="main-content">
```
Comment (aliaksandrkrasnou) [inline index.html]: If it's a main-content block, maybe main tag could be more useful instead of div?

File: menu.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+nav ul{
+    display: flex;
+    height: 100%;
+    width: 100%;
+    margin: 0px;
```
Comment (aliaksandrkrasnou) [inline menu.css]: No need to put down px if you set value to 0

Comment (eqxBarboss) [reply menu.css]: Same to padding?

**Comment by aliaksandrkrasnou** [comment]
Approved 👍

**Pull request closed without merging**